packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/generate-auth-trigger-template.ts

```diff
@@ -75,10 +89,15 @@ export async function generateNestedAuthTriggerTemplate(context: $TSContext, cat
   }
 }
 
-async function createCustomResourceforAuthTrigger(context: any, authTriggerConnections: AuthTriggerConnection[]) {
+async function createCustomResourceforAuthTrigger(
+  context: any,
+  authTriggerConnections: AuthTriggerConnection[],
+  permissions: AuthTriggerPermissions[],
+) {
   const stack = new CustomResourceAuthStack(undefined as any, 'Amplify', {
     description: 'Custom Resource stack for Auth Trigger created using Amplify CLI',
     authTriggerConnections: authTriggerConnections,
+    permissions: permissions,
```

Comment:
no need for `authTriggerConnections` and `permissions` after the colon as the names are the same.